src/controller/CHIPDeviceController.cpp

```diff
@@ -1090,6 +1090,18 @@ void DeviceCommissioner::OnFailedToExtendedArmFailSafeDeviceAttestation(void * c
     commissioner->CommissioningStageComplete(CHIP_ERROR_INTERNAL, report);
 }
 
+void DeviceCommissioner::ExtendArmFailSafe(DeviceProxy * proxy, CommissioningStage step, uint16_t armFailSafeTimeout,
```

Comment:
Should `DeviceCommissioner::PerformCommissioningStep` in the `kArmFailsafe` just be calling this function?  Seems like it should be.  It can VerifyOrDie that its `endpoint` is `kRootEndpointId`.

Reply:
fixed to call this function